### PR TITLE
Add issue template to get more complete issues

### PR DESCRIPTION
**Pull request by prometheanfire**
Add an issue template in the .github folder as described
https://github.com/blog/2111-issue-and-pull-request-templates

A list of questions has been added so that hopefully more information
is not needed when prioritizing, targeting, and working on issues.

Fixes: #855

### CODE REVIEW

**Comment by sigmavirus24** [comment]
Preliminary :+1: but someone should try this out on a multinode. :trollface:

**Comment by prometheanfire** [comment]
lol

**Comment by prometheanfire** [comment]
recheck - ssh timeout

    21:41:50 failed: [jrpcaioiad-8b3_utility_container-53e20f05] => {"failed": true, "parsed": false}

**Comment by prometheanfire** [comment]
recheck - tempest

    15:46:35     <class 'neutronclient.common.exceptions.PortNotFoundClient'>

**Comment by d34dh0r53** [comment]
The checklist at the bottom should include a dummy link to the PR as is in the contributing docs.  Other than that it looks good.

**Comment by d34dh0r53** [comment]
Nevermind, I'm an idiot

**Comment by prometheanfire** [comment]
this is for issue creation, before PRs are created, so there will be no links available yet

**Comment by d34dh0r53** [comment]
:+1:

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+##### If this is a large issue, what can be done immediately to alleviate the issue while working on a longer-term solution?
+
+##### What is your evaluation on the importance of this issue, and why?
+
+##### What branches / milestones should this issue target?
+The list below is what is currently supported for backports, remove branches you do not wish to target.
+- [ ] Master
+- [ ] liberty-12.0
+- [ ] kilo
```
Comment (mancdaz) [inline .github/ISSUE_TEMPLATE.md]: should the reporter get to decide this? Isn't this down to the triage process, and the core/PTL team to decide, based on stable branch process?

Comment (prometheanfire) [inline .github/ISSUE_TEMPLATE.md]: it's not spelled out here, but this is just gathering what they wish to target, not what is being targeted.

Comment (prometheanfire) [inline .github/ISSUE_TEMPLATE.md]: perhaps changing the question to `What branches / milestones do you believe this issue should target?`

Comment (d34dh0r53) [inline .github/ISSUE_TEMPLATE.md]: That is better wording @prometheanfire

Comment (mancdaz) [inline .github/ISSUE_TEMPLATE.md]: I guess it's just like someone in launchpad adding *-backport-potential tags - they can always be removed later by the triage/reviewer/maintainer team. So yeah, maybe a tweak to the wording but other than that looks good.

**Comment by mancdaz** [comment]
:+1:  pending gate

**Comment by d34dh0r53** [comment]
recheck - rabbit - for an MD file - facepalm

**Comment by prometheanfire** [comment]
recheck - ceph timeout

    18:20:56 Build timed out (after 20 minutes). Marking the build as aborted.

**Comment by tomc603** [comment]
:+1: And since this makes 3, I'm merging.

**Pull request merged**